runtime/astar/src/lib.rs

```diff
@@ -391,10 +391,11 @@ impl pallet_session::Config for Runtime {
 
 parameter_types! {
     pub const PotId: PalletId = PalletId(*b"PotStake");
-    pub const MaxCandidates: u32 = 200;
+    pub const MaxCandidates: u32 = 148;
     pub const MinCandidates: u32 = 5;
-    pub const MaxInvulnerables: u32 = 20;
+    pub const MaxInvulnerables: u32 = 48;
     pub const SlashRatio: Perbill = Perbill::from_percent(1);
+    pub const KickThreshold: BlockNumber = 2 * SessionPeriod;
```

Comment:
Since kick-threshold has been increased, should the slashing amount be doubled too?

Reply:
Worst thing collator can do is skip a block, but increasing collator set will mininize damage. Finally we plan to have same block production window as we have now.